Modulefile

```diff
@@ -7,6 +7,6 @@ summary 'Minecraft / CraftBukkit Server v2'
 description 'Install and manage a Minecraft or CraftBukkit server'
 project_page 'https://github.com/andschwa/puppet-minecraft'
 
-dependency 'maestrodev/wget', '>= 1.1.0'
+dependency 'puppet/archive', '>= 0.5.1'
```

Comment:
I guess requiring 1.0.0 is fine so long as a new release of this module isn't released before 1.0.0 of puppet-archive hits the forge.

As for the version of this module, it's between 2.2.0 or 3.0.0.  @bastelfreak - thoughts?  This module itself won't be ruby 1.8 incompatible (no functions, types, or facts), but with the recommended puppet-archive dependency of 1.0.0, it'd effectively would be.

So, if you go with depending on puppet-archive 1.0.0, go with 3.0.0, otherwise 2.2.0 should be ok???